src/emm/services/git_service.py

```diff
@@ -146,6 +146,31 @@ def commit_all(self, commit_message: str, directory: Optional[Path] = None) -> N
         with local.cwd(self._determine_directory(directory)):
             self.git[args]()
 
+    def pull_request(self, extra_args: List[str], directory: Optional[Path] = None) -> str:
```

Comment:
We might want to consider a separate service for working with the github command. So far we have kept each service working with a single command. Now this service is using two different commands. That might get confusing to people looking at it in the future.